packages/commerce-sdk-react/tsconfig.json

```diff
@@ -12,7 +12,7 @@
 
         /* Language and Environment */
         "target": "es2016" /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */,
-        // "lib": [],                                        /* Specify a set of bundled library declaration files that describe the target runtime environment. */
+        "lib": ["dom", "es2019"],                                        /* Specify a set of bundled library declaration files that describe the target runtime environment. */
```

Comment:
If we have `target: es2016`, should we also use the `es2016` lib, so that we only use available features? Or are we fine with the compiler inserting things that we need?

Reply:
From what I read there is a kind of chaining that would be happening es2019 uses es2018 and it goes down the line. Anyway. I needed to add this to get the deep clone inplementation working using the `entries` method.

Reply:
I'm fairly confident that this is fine from a "code that compiles and runs" standpoint. The thing I'm not sure about is how the es2019 code gets changed to es2016 code, particularly the impact on bundle size. Might be worth investigating as a separate ticket. (Outcome would probably either be "this is fine" or "don't use modern code so we don't add polyfill bloat".)